syndic.opam

```diff
@@ -7,20 +7,25 @@ license:      "MIT"
 homepage:     "https://github.com/Cumulus/Syndic"
 dev-repo:     "git+https://github.com/Cumulus/Syndic.git"
 bug-reports:  "https://github.com/Cumulus/Syndic/issues"
+synopsis:     "RSS1, RSS2, Atom and OPML1 parsing"
+description: """Pure OCaml Library for parsing and writing various types of
```

Comment:
I think it is nice to have a blank line between the synopsis and the description.  Hence:
```
description: """
Pure OCaml...
```